Sources/OktaDirectAuth/DirectAuthFlow.swift

```diff
@@ -22,8 +22,15 @@ public protocol DirectAuthenticationFlowDelegate: AuthenticationDelegate {
 }
 
 public enum DirectAuthenticationFlowError: Error {
+    case invalidOTP
+    case oobRejected
+    case invalidChallengeTypesSupported
+    case pollingTimeoutExceeded
     case missingArguments(_ names: [String])
-    case currentStatusMissing
+    case network(error: APIClientError)
```

Comment:
In push rejected scenario I got a `.network(error:)`
![Screen Shot 2023-03-14 at 2 08 13 PM](https://user-images.githubusercontent.com/77817162/225137864-4f91ed07-060d-4680-8ec9-7d25101e1bbf.png)